object_tracker_0/src/inference.py

```diff
@@ -57,7 +57,8 @@ def convert_bbox_format_2(bbox, shape):
     x0, y0, x1, y1 = bbox
     return [ x0, y0, x1 - x0, y1 - y0 ]
 
-def is_similar(box1, box2, threshold=10):
+# TO DO: probably we should use IoU to be more precise.  E.g. using an absolute threshold, it could be too lenient for small objects and too strict for big ones.
```

Comment:
I agree. We should move the IoU calc to an utils file probably and reuse from there.

Reply:
I'll do this in a separate PR